topics/introduction/tutorials/galaxy-cite/tutorial.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: tutorial_hands_on
+title: Best Practices for Citing Galaxy
+tags:
+  - fair
+  - introduction
+level: Introductory
+questions:
+  - How to cite Galaxy in Publications?
+objectives:
+  - Understand how to cite Galaxy.
+  - Prepare your methodology section based on your Galaxy Histories.
+time_estimation: 10m
```

Comment:
I would increase the time, maybe to 15 or 20 minutes. I know there are not really many hands-on parts, but there are some links that could also take time to go over.